instrumentation/kafka-clients/kafka-clients-0.11/javaagent/build.gradle.kts

```diff
@@ -19,25 +21,13 @@ dependencies {
 
   library("org.apache.kafka:kafka-clients:0.11.0.0")
 
-  testLibrary("org.springframework.kafka:spring-kafka:1.3.3.RELEASE")
-  testLibrary("org.springframework.kafka:spring-kafka-test:1.3.3.RELEASE")
-  testImplementation("javax.xml.bind:jaxb-api:2.2.3")
-  testLibrary("org.assertj:assertj-core")
-
-  // Include latest version of kafka itself along with latest version of client libs.
-  // This seems to help with jar compatibility hell.
-  latestDepTestLibrary("org.apache.kafka:kafka_2.11:2.3.+")
-  // (Pinning to 2.3.x: 2.4.0 introduces an error when executing compileLatestDepTestGroovy)
-  //  Caused by: java.lang.NoClassDefFoundError: org.I0Itec.zkclient.ZkClient
-  latestDepTestLibrary("org.apache.kafka:kafka-clients:2.3.+")
-  latestDepTestLibrary("org.springframework.kafka:spring-kafka:2.2.+")
-  latestDepTestLibrary("org.springframework.kafka:spring-kafka-test:2.2.+")
-  // assertj-core:3.20.0 is incompatible with spring-kafka-test:2.7.2
-  latestDepTestLibrary("org.assertj:assertj-core:3.19.0")
+  testImplementation("org.testcontainers:kafka:${versions["org.testcontainers"]}")
```

Comment:
I think we better add this dependency into corresponding dependencySet in dependencyManagement module.

Reply:
AFAIR we're using testcontainers' additional libs this way everywhere. Do you mind if I fix that everywhere in the next PR?

Reply:
As this fixes build if would be best to merge this asap and deal with minor issues such as this in a subsequent pr.